substrafl/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.37.0"
+__version__ = "0.38.0"
```

Comment:
```suggestion
__version__ = "0.38.0rc1"
```

Reply:
Pre-release will be based on [release-0.38.0-rc1](https://github.com/Substra/substrafl/tree/release-0.38.0-rc1). Is that bad?

Reply:
Nop, It's just that usually we iterate on the same branch, but this is as valid as the other method :)